includes/wikia/api/DWDimensionApiController.class.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+class DWDimensionApiController extends WikiaApiController {
+	const LIMIT = 100;
+	const AFTER_DOMAIN = null;
+
+	private function getSharedDbSlave() {
+		global $wgExternalSharedDB;
+		return wfGetDB( DB_SLAVE, array(), $wgExternalSharedDB );
+	}
+
+	public function getWikiDomains() {
+		$limit = $this->getRequest()->getVal( 'limit', static::LIMIT );
+		$afterDomain = $this->getRequest()->getVal( 'after_domain', static::AFTER_DOMAIN );
+
+		$db = $this->getSharedDbSlave();
+		$dbResult = $db->select(
+			[ 'city_domains' ],
+			[ 'city_id', 'city_domain' ],
+			isset( $afterDomain ) ? [ 'city_domain > '.$afterDomain ] : [ ],
```

Comment:
is this ok or do I need to add some explicit escaping to avoid sql injection? Or is it handled by getRequest()->getVal?

Reply:
Nope, you need to call `$db->strencode`

Reply:
Ok, added it